src/validationinterface.cpp

```diff
@@ -177,18 +177,17 @@ void SyncWithValidationInterfaceQueue()
 // evaluating arguments when logging is not enabled.
 //
 // NOTE: The lambda captures all local variables by value.
-#define ENQUEUE_AND_LOG_EVENT(event, fmt, name, ...)           \
-    do {                                                       \
-        auto local_name = (name);                              \
-        LOG_EVENT("Enqueuing " fmt, local_name, __VA_ARGS__);  \
-        m_internals->m_schedulerClient.AddToProcessQueue([=] { \
-            LOG_EVENT(fmt, local_name, __VA_ARGS__);           \
-            event();                                           \
-        });                                                    \
+#define ENQUEUE_AND_LOG_EVENT(category, level, event, fmt, ...)    \
+    do {                                                           \
+        LOG_EVENT(category, level, "Enqueuing " fmt, __VA_ARGS__); \
+        m_internals->m_schedulerClient.AddToProcessQueue([=] {     \
+            LOG_EVENT(category, level, fmt, __VA_ARGS__);          \
+            event();                                               \
+        });                                                        \
     } while (0)
 
-#define LOG_EVENT(fmt, ...) \
-    LogPrint(BCLog::VALIDATION, fmt "\n", __VA_ARGS__)
+#define LOG_EVENT(category, level, fmt, ...) \
+    LogPrintLevel(category, level, fmt "\n", __VA_ARGS__)
```

Comment:
There's not really any need for a separate `LOG_EVENT` macro anymore -- all it saves now is adding a newline.

Reply:
Got rid of it.